draft-ietf-quic-qpack.md

```diff
@@ -1225,12 +1225,12 @@ a linear-time attack.
 ## Applicability to QPACK and HTTP
 
 QPACK mitigates but does not completely prevent attacks modeled on CRIME [CRIME]
-by forcing a guess to match an entire header field value, rather than individual
+by forcing a guess to match an entire field value, rather than individual
```

Comment:
i think it has to match a field line, not field value (here and below)?  And technically the original HPACK text should have said entire header (you have to match the name and value).